GitUI/Editor/FileViewerInternal.cs

```diff
@@ -246,6 +302,18 @@ public void SetText(string text, Action openWithDifftool, bool isDiff = false)
             TextEditor.Refresh();
 
             _currentViewPositionCache.Restore(isDiff);
+
+            if (_shouldScrollToBottom)
+            {
+                if (TextEditor.ActiveTextAreaControl.VScrollBar.Visible)
+                {
+                    TextEditor.ActiveTextAreaControl.VScrollBar.Value =
+                        Math.Max(0, TextEditor.ActiveTextAreaControl.VScrollBar.Maximum -
+                        TextEditor.ActiveTextAreaControl.VScrollBar.Height - _bottomBlankHeight);
```

Comment:
Ditto, keep on the same line
```suggestion
                        Math.Max(0, TextEditor.ActiveTextAreaControl.VScrollBar.Maximum - TextEditor.ActiveTextAreaControl.VScrollBar.Height - _bottomBlankHeight);
```

Reply:
ok